src/uwtools/drivers/shave.py

```diff
@@ -41,9 +41,7 @@ def provisioned_run_directory(self):
         Run directory provisioned with all required content.
         """
         yield self._taskname("provisioned run directory")
-        yield [
-            self.runscript(),
-        ]
+        yield self.runscript()
```

Comment:
It's ok for a `@tasks` method to return a single task it depends on, instead of a one-item collection.

Reply:
Thats fair - thanks!